README.md

```diff
@@ -137,6 +132,49 @@ Analytics.setGlobalProperties({
 
 Note that global properties can be overridden by parameters passed to `trackEvent` calls.
 
+### Initializing `infermedicaAnalytics` module
+#### In order to use `infermedicaAnalytics` module you must initialize it first.
+
+You can decide if you want to use Firebase **anonymously** or as **identified user**. In order to use `infermedicaAnalytics` 
+module of this package you should call `initialize()` method and pass it an object which matches the following type:
+
+```typescript
+type InitializeParams = IAnonymousInitializeParams | IAuthenticatedInitializeParams;
+
+interface IAnonymousInitializeParams {
+  firebaseConfig: FirebaseOptions,
+  forceSignInAnonymously: true,
+}
+
+interface IAuthenticatedInitializeParams {
+  firebaseAuth: Auth,
+  forceSignInAnonymously: false,
+}
+```
+
+If you want to let the library sign you in to firebase as anonymous user, you should call the `initialize()` method with an object corresponding to the `IAnonymousInitializeParams` `interface`, e.g.:
+
+```javascript
+import { Analytics } from '@infermedica/modular-analytics';
+
+const firebaseConfig = { /* your firebase config */ };
+Analytics.initialize({ firebaseConfig, forceSignInAnonymously: true });
```

Comment:
Q: where you use `firebaseConfig` in module line 260 still has `initializeApp({})` with empty object instead `firebaseConfig`

Reply:
done in  [b11c709](https://github.com/infermedica/modular-analytics/pull/5/commits/b11c7093d16c07fe3c5b080aafde8ce54b738ffa)  :)